test/input_json/FinancialModels/equpartflip/2022.08.08_develop_branch_Geothermal_Power_All_Equity_Partnership_Flip_cmod_equpartflip.json

```diff
@@ -192,6 +192,7 @@
     "dispatch_factor7" : 1,
     "dispatch_factor8" : 1,
     "dispatch_factor9" : 1,
```

Comment:
Should we be clearing the old dispatch factors out of the test data? Is it best practice to reexport these for this type of change?

Reply:
Best practices would probably be to tie to latest release and then re-export - what do you think?